deploy/crds/kustomize_operatorpolicy/policy.open-cluster-management.io_operatorpolicies.yaml

```diff
@@ -68,68 +68,66 @@ spec:
                     type: string
                   target:
                     description: Target namespaces of the referent
-                    items:
-                      properties:
-                        namespaces:
-                          description: '''namespaces'' and ''selector'' both define
-                            an array/set of target namespaces that should be affected
-                            on the cluster. Only one of ''namespaces'' or ''selector''
-                            should be specified, and if both are set then ''selector''
-                            will be omitted.'
-                          items:
-                            type: string
-                          type: array
-                        selector:
-                          description: '''namespaces'' and ''selector'' both define
-                            an array/set of target namespaces that should be affected
-                            on the cluster. Only one of ''namespaces'' or ''selector''
-                            should be specified, and if both are set then ''selector''
-                            will be omitted.'
-                          properties:
-                            matchExpressions:
-                              description: matchExpressions is a list of label selector
-                                requirements. The requirements are ANDed.
-                              items:
-                                description: A label selector requirement is a selector
-                                  that contains values, a key, and an operator that
-                                  relates the key and values.
-                                properties:
-                                  key:
-                                    description: key is the label key that the selector
-                                      applies to.
-                                    type: string
-                                  operator:
-                                    description: operator represents a key's relationship
-                                      to a set of values. Valid operators are In,
-                                      NotIn, Exists and DoesNotExist.
+                    properties:
+                      namespaces:
+                        description: '''namespaces'' and ''selector'' both define
```

Comment:
Future TODO: I don't find the "both" statement helpful (and more confusing than anything). This needs to only describe "namespaces", leaving in the last sentence that only one can be specified. Why add a note though, rather than adding a `oneOf` requirement to the namespaces/selector?

Reply:
You're absolutely right, but we directly embed these types as defined by OLM, so this documentation and the way it works is from them.

Reply:
> You're absolutely right, but we directly embed these types as defined by OLM, so this documentation and the way it works is from them.

Ah, that makes sense. Bummer. 😕